src/textures/Texture.js

```diff
@@ -70,7 +70,8 @@ class Texture extends EventDispatcher {
 		this.version = 0;
 		this.onUpdate = null;
 
-		this.isRenderTargetTexture = false;
+		this.isRenderTargetTexture = false; // indicates whether a texture belongs to a render target or not
+		this.needsPMREMUpdate = false; // indicates whether this texture should be processed by PMREMGenerator or not (only relevant for render target textures)
```

Comment:
I am inclined to think these properties should be "private", since they are for renderer-use only.
```js
this._isRenderTargetTexture = false;
this._needsPMREMUpdate = false;
```

Reply:
Agreed 👍

Reply:
edit: I'm assuming it should be a double-underscore.

Reply:
Depending on how user or third-party code is developed, both flags are actually required. 

`isRenderTargetTexture` is just the pendant to all other `is*` properties and required for type checks. Besides, users have to set `needsPMREMUpdate` to `true` (similar to other `needsUpdate` flags) if they are using a dynamic render target as an env map applied to PBR materials and author the render target's data by themselves.

Reply:
I was actually thinking about suggesting something like `RenderTargetTexture`. However, this idea is not yet mature.

Reply:
Okay, lets test performance in the meantime.